### PR TITLE
Test restore on DR site works

### PR DESCRIPTION
**Pull request by KillianG**
Reactivate restore on DR site tests

### CODE REVIEW

File: tests/ctst/steps/utils/kubernetes.ts [hunk 1]
```diff
@@ -63,6 +63,22 @@ export function createKubeCustomObjectClient(world: Zenko) {
...
 }
 
+export async function deletePodsByLabel(world: Zenko, label: string, namespace = 'default') {
+    const coreClient = createKubeCoreClient(world);
+
+    try {
+        await coreClient.deleteCollectionNamespacedPod(
+            namespace, undefined, undefined, undefined, undefined, undefined, label);
+    } catch (err) {
+        world.logger.error('Error deleting pods by label', {
```
Comment (ghost) [inline tests/ctst/steps/utils/kubernetes.ts]:
```suggestion
        world.logger.debug('Error deleting pods by label', {
```

Error logs will dump all the logs in-flight for the "request", here the scenario; but we already do it at the end of any scenario if the result is "failed"

Comment (ghost) [reply tests/ctst/steps/utils/kubernetes.ts]: (I know we have "error" in other places)

**Review by ghost — APPROVED** [review]

File: tests/ctst/steps/pra.ts [hunk 2]
```diff
@@ -71,6 +72,7 @@ async function installPRA(world: Zenko, sinkS3Endpoint = 'http://s3.zenko.local'
         // prometheusHostname: 'prom.dr.zenko.local', // could be any name, cert will be auto-generated
         prometheusExternalIpsDiscovery: true,
         prometheusDisableTls: true,
+        forceRotateServiceCredentials: true,
```
Comment (francoisferrand) [inline tests/ctst/steps/pra.ts]: should not be set on "initial" install : only use on reinstall
(and un-related to this PR)

Comment (francoisferrand) [reply tests/ctst/steps/pra.ts]: btw should just be removed, this is unrelated to this PR and will conflict with one about `uninstall`

**Review by francoisferrand — APPROVED** [review]

**Comment by francoisferrand** [comment]
Merged in #2136

**Pull request closed without merging**